### DISCUSSION TITLE
Unicode/Chinese characters on My Media

### DISCUSSION THREAD

**Discussion by explicitworkload** [jellyfin/jellyfin#9286] [Q&A] [answered]
Hi Jellyfin,

This issue came up after a couple of days of adding new libraries, and I can't seem to get unicode/chinese characters showing up in the overlay/banner. Refer to the image. Non-english characters shows up in boxes. How do I get non-english characters to show up properly?

Any idea how this can be done?

**Comment by nyanmisaka** [comment, marked as answer]
There's no CJK fonts installed on your system. On Debian/Ubuntu: `sudo apt install fonts-noto-cjk -y` and refresh the thumbnail.

**Reply by explicitworkload** [reply]
Thank you. I searched in Discussions and couldn't find any discussions on this point. But I managed to find others who have similar issues. ;)

Edit:
Managed to get CJK to work in docker by doing the following:-
1. download CJK fonts from [Google](https://fonts.google.com/noto/use)
2. Mount it as a volume to /usr/share/fonts/ (i copied out dejavu as well)

```
     volumes:
      - ./fonts:/usr/share/fonts
```      
All the CJK fonts should reside in your PV (./fonts)
